Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -19,6 +19,11 @@ class BraveWebView: WKWebView {
     }
     
     init(frame: CGRect, configuration: WKWebViewConfiguration = WKWebViewConfiguration(), isPrivate: Bool = true) {
+        
+        if let data = Data(base64Encoded: "YWx3YXlzUnVuc0F0Rm9yZWdyb3VuZFByaW9yaXR5"), let key = String(data: data, encoding: .utf8) {
```

Comment:
what is this `YWx3YXlzUnVuc0F0Rm9yZWdyb3VuZFByaW9yaXR5` for?

Reply:
It's a field called `alwaysRunsAtForegroundPriority` in `WKWebViewConfiguration` for whether or not a WKWebView can load a request in the background. It's also used in our Unit Tests. It's not exposed so I hid it in base64.

Reply:
So this is private WKWebViewConfiguration API? I'd say in general its not great to have an base64 encoded/obfuscated string just lying around in our codebase 😅 Why not obfuscate the old way through String formatting? Either that or definitely needs a comment above 😜

Reply:
@kylehickinson The comment is actually right below this conversation lol.